pkg/crc/image/image.go

```diff
@@ -112,9 +113,16 @@ func PullBundle(preset crcpreset.Preset) error {
 	if err != nil {
 		return err
 	}
-	_, err = extract.Uncompress(filepath.Join(destDir, imgLayer), constants.MachineCacheDir, true)
+	fileList, err := extract.Uncompress(filepath.Join(destDir, imgLayer), constants.MachineCacheDir, true)
 	if err != nil {
 		return err
 	}
+	logging.Debugf("Bundle and sign path %v", fileList)
+
+	logging.Info("Verifying the bundle signature...")
+	if err := gpg.Verify(fileList[0], fileList[1]); err != nil {
```

Comment:
I would not hardcode that first file in the archive will always be the bundle, and second file its signature.

Reply:
@cfergeau this is the way our script will create bundle, should we compare the size or with the extention?

Reply:
It is now compare with extension so should be good.